frontend/app/views/joiner/form/monthlyContribution.scala.html

```diff
@@ -58,11 +58,11 @@
         }());
     </script>
 
-    <main class="page-content skin-checkout-b skin-checkout-paypal">
+    <main class="page-content skin-checkout-b skin-checkout-paypal skin-monthly-contribution">
```

Comment:
Yes, the `skin-monthly-contribution` loads the exception assuming that there are the other two skins applied. A new design will be introduced soon and probably we can drop these skins from this form.